<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QAB7R0C" failures="1" tests="8" name="Test" time="50.589" errors="0" timestamp="2022-08-19T15:24:43 IST">
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="createClassTest" time="3.617"/>
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="SubmitAttendanceTest" time="10.201"/>
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="CheckAttendanceOfStudentTest" time="3.602"/>
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="VerifyAttendance" time="4.684">
    <skipped/>
  </testcase> <!-- VerifyAttendance -->
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="VerifyAttendance" time="4.614">
    <skipped/>
  </testcase> <!-- VerifyAttendance -->
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="VerifyAttendance" time="4.626">
    <skipped/>
  </testcase> <!-- VerifyAttendance -->
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="VerifyAttendance" time="3.382">
    <skipped/>
  </testcase> <!-- VerifyAttendance -->
  <testcase classname="com.AttendanceManagement.Dashboard.AttendanceManagement" name="VerifyAttendance" time="4.707">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.AttendanceManagement.Dashboard.AttendanceManagement.VerifyAttendance(AttendanceManagement.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAttendance -->
</testsuite> <!-- Test -->
